cinema_service/urls.py

```diff
@@ -2,10 +2,22 @@
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import path, include
+from drf_spectacular.views import (
+    SpectacularAPIView,
+    SpectacularSwaggerView,
+    SpectacularRedocView
+)
 
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
```

Comment:
The inclusion of the `cinema` app's URLs under the `api/cinema/` path is appropriate. Ensure that the `cinema.urls` module is correctly configured with the necessary URL patterns.